src/SourceGenerators/Uno.UI.SourceGenerators/Telemetry/TelemetryCommonProperties.cs

```diff
@@ -69,14 +82,24 @@ private string GetMachineId()
 					select nic.GetPhysicalAddress().ToString()
 				).FirstOrDefault();
```

Comment:
You should have a .OrderBy() to ensure you're always taking the same one.

Reply:
It's already ordered by interface weight, as defined by the underlying APIs.